src/wowman/tukui_api.clj

```diff
@@ -21,12 +21,15 @@
   [addon-summary game-track]
   (let [url (format (if (= game-track "classic") classic-summary-url summary-url)
                     (:source-id addon-summary))
-        ti (some-> url http/download utils/from-json spy)
-        ]
-    (merge addon-summary
-           {:download-uri (:url ti)
-            :version (:version ti)
-            :interface-version (-> ti :patch utils/game-version-to-interface-version)})))
+
+        ;; tukui will return a successful but-empty response (200) for addons that
+        ;; exist but not for the requested game track.
```

Comment:
Not that it changes the logic from what I can tell but Tukui will respond with an empty 200 for add-ons that don't exist whether 'they' exist in the opposite track or not. (And I say 'they' because IDs are not shared across tracks.)

Reply:
> And I say 'they' because IDs are not shared across tracks.

Thank you for pointing that out. I created two new sources "tukui" and "tukui-classic" to handle it.

I had completely forgotten about it though.

I should make it obvious what the code is doing, because the natural assumption is there is one catalogue and one set of IDs. 

I could join the two catalogues up, create a synthetic ID with classic and retail 'releases', but nah. Too much effort.

Reply:
improved comment